### PR TITLE
upgrade to angular beta 0

### PR DESCRIPTION
**Pull request by diegoconcha**

### CODE REVIEW

File: src/app/boot.ts [hunk 1]
```diff
@@ -0,0 +1,16 @@
+import {bootstrap} from 'angular2/platform/browser'
+import {AppComponent} from './component'
+import {Api} from '../features/measurements/api';
+
+// Why use a seperate boot.ts file?
```
Comment (kswin) [inline src/app/boot.ts]: This is an awesome explanation

File: src/index.html [hunk 2]
```diff
@@ -1,16 +1,26 @@
...
     <script src="../node_modules/angular2/bundles/angular2.dev.js"></script>
+
+    <!-- 2. Configure SystemJS -->
     <script>
       System.config({
         packages: {
-          'app': {defaultExtension: 'js'},
-          'features': {defaultExtension: 'js'}
+          app: {
+            format: 'register',
```
Comment (kswin) [inline src/index.html]: what does this do?

**Comment by kswin** [comment]
Looks good!!! :rocket:

File: src/index.html (hunk 2 above)
Comment (diegoconcha) [inline src/index.html]: It specifies the module format. I'll add a comment.

https://github.com/systemjs/systemjs/blob/master/docs/module-formats.md#module-format-detection

https://github.com/ModuleLoader/es6-module-loader/blob/v0.17.0/docs/system-register.md

**Pull request merged**